src/com/estafet/learning/US008/TradeOrder.java

```diff
@@ -0,0 +1,15 @@
+package com.estafet.learning.US008;
+
+public class TradeOrder extends Order {
+    final String color = "red";
+
+    public TradeOrder() {
+
+    }
+
+    public void tradeOrder() {
+        System.out.println(color);//prints color of Dog class
+        System.out.println(Order.color);//prints color of Animal class
```

Comment:
are you sure you need static value (Order.color) here instead of parent's instance value (super.color)? Though won't make any difference unless you change its value dynamically :)

Reply:
please explain the difference between:
`Order.color` and `super.color`

Reply:
super refers to parent class. 
with Order is pointing specific class